pkg/server/server_authz_test.go

```diff
@@ -700,6 +707,80 @@ func TestCheckAuthz(t *testing.T) {
 	})
 }
 
+func TestGetAccessibleStores(t *testing.T) {
+	t.Cleanup(func() {
+		goleak.VerifyNone(t)
+	})
+	ds := memory.New()
+	t.Cleanup(ds.Close)
+
+	t.Run("check_no_authz", func(t *testing.T) {
```

Comment:
nitpik: check_no_authz reminds me of "check" as in OpenFGA.  Perhaps avoid the word "check"?